app/assets/javascripts/discourse/app/templates/components/tag-info.hbs

```diff
@@ -1,16 +1,41 @@
 <section class="tag-info">
   {{#if tagInfo}}
     <div class="tag-name">
-      {{discourse-tag tagInfo.name tagName="div" size="large"}}
+      {{#if editing}}
+        <div class="edit-tag-wrapper">
```

Comment:
If we were really being pedantic maybe we could put this into a `discourse-tag-edit-form` component...maybe in future if we add more properties to edit?